lib/core/empty.dart

```diff
@@ -2,9 +2,9 @@ import 'package:flutter/material.dart';
 
```

Comment:
1. semua kode widget ini taruh dalam folder `core/widgets/ `
2. tiap file widgetnya, kasih `part of 'package:ecowave/core.dart';`. jadi gaada import lagi.
3. jangan lupa di file `lib/core.dart` tambahin `part 'package:ecowave/core/widgets/empty.dart';` dan lainnya
4. color disamain dengan kode warna di figma, cara nya `color: Color(0xff######)`